estrategiacontenidos/mapadelsitio.md

```diff
@@ -1,16 +1,21 @@
 # Mapa del sitio
 
-<!--
-Recoged el mapa del sitio en este documento en forma de lista
-formateada usando la sintaxis de Markdown, así:
-
-- si queréis
-- una lista
-  + sin ordenar
-
-o así:
-
-1. si queréis
-   1. una lista
-   2. ordenada
--->
+1. Página principal
+    1. **Podcast**
+         1. Noticiario 1
+             - Audio noticiario 1
+         2. Noticiario 2
+             - Audio noticiario 2
+         3. Noticiario 3
+              - Audio noticiario 3
+         4. Tomas Falsas
+              - Lista de Vídeos
+    2. **Revista**
+         1. "Política y Pipas"
+             - Audios de las entrevistas
```

Comment:
Me sigue sin quedar claro tampoco en este caso. "Política y pipas" es una página sobre la revista, y luego hay otra página con los "audios de las entrevistas"? Esos audios son material complementario a las entrevistas que publicáis en la revista, supongo, pero ¿es el único tipo de material complementario que vais a incluir?